.ahoy/site/.scripts/site.files-fix-permissions.sh

```diff
@@ -7,3 +7,10 @@ if [ -d docroot/sites/default/files ]; then
   find docroot/sites/default/files/ -type d -exec chmod o+rwx {} \;
   find docroot/sites/default/files/ -type f -exec chmod o+rw {} \;
 fi
+
+mkdir -p docroot/sites/default/private
```

Comment:
I think this line was left over from development process, otherwise the next line is never needed.